source/adapter.screenshare.js

```diff
@@ -75,9 +74,9 @@
     };
 
   } else if (window.navigator.webkitGetUserMedia) {
-    baseGetUserMedia = window.navigator.getUserMedia;
 
     navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      baseGetUserMedia = window.navigator.getUserMedia;
```

Comment:
This causes `Maximum call stack size exceeded` error (an infinite loop). I would suggest moving back outside the scope of `navigator.getUserMedia`.